docs/pywbemcli/commands.rst

```diff
@@ -2288,7 +2288,7 @@ The ``connection show`` command shows information about a connection definition:
 
 .. code-block:: text
 
-    pywbemcli -s http://blah connection show
+    pywbemcli> -s http://blah connection show
```

Comment:
If you want to show the interactive prompt, you should also show the command to get there, like in the `connection select` command, above.

Reply:
Fixed a bunch of them.